packages/npm-dependencies-validation/src/types.ts

```diff
@@ -1,19 +1,4 @@
-export type NpmLsResult = {
-  problems: string[];
-};
-
-export type DependenciesPropertyName = "dependencies" | "devDependencies";
-
-export type PackageJson = {
-  name: string;
-  version: string;
-} & DependenciesProperties;
-
-export type DependenciesProperties = {
-  [key in DependenciesPropertyName]?: {
-    [key: string]: string;
-  };
-};
+import { PackageJson } from "type-fest";
```

Comment:
"type-fest" is very nice for missing "built-in" types
- https://github.com/sindresorhus/type-fest